1.segment_images/scripts/0.segment_nuclei_organoids.py

```diff
@@ -61,8 +61,10 @@
     input_dir = pathlib.Path(args.input_dir).resolve(strict=True)
 else:
     print("Running in a notebook")
-    input_dir = pathlib.Path("../../data/z-stack_images/C4-2/").resolve(strict=True)
-    window_size = 3
+    input_dir = pathlib.Path("../../data/NF0014/zstack_images/C4-2/").resolve(
+        strict=True
+    )
+    window_size = 5
```

Comment:
Looks like this window size has changed, consider adding a reasoning if applicable.